Text/CustomEntitySearch/WordLinker.cs

```diff
@@ -14,13 +15,33 @@ internal class WordLinker
         public WordLinker(string executingDirectoryPath)
         {
             string json = File.ReadAllText($"{executingDirectoryPath}\\words.json");
-            Words = new List<string>(
-                JsonConvert.DeserializeObject<List<string>>(json));
+            WordLinker convert = JsonConvert.DeserializeObject<WordLinker>(json);
+            Words = convert.Words;
+            Synonyms = convert.Synonyms;
+            ExactMatch = convert.ExactMatch;
+            FuzzyMatchOffset = convert.FuzzyMatchOffset;
+            CaseSensitive = convert.CaseSensitive;
         }
 
         public IList<string> Words
         {
             get; private set;
         }
+        public IList<string> ExactMatch
+        {
+            get; private set;
```

Comment:
No need to specify `private set`: C# allows you to set get-only properties from your constructor. You only need this if you set those properties from anywhere else.